### DISCUSSION TITLE
Jazzer for servlets

### DISCUSSION THREAD

**Discussion by hector-rojas-sci** [CodeIntelligenceTesting/jazzer#245] [General]
Hi, you have an interesting project, I'm starting to work with this project and I have a question, is it possible to use it to test Servlets?? I hope you can help me, I don't have an idea about how do it.

**Comment by fmeum** [comment]
The general approach to fuzz servlets with Jazzer would be to wrap the controllers into fuzz targets that supply the required parameters, e.g. using the [FuzzedDataProvider](https://github.com/CodeIntelligenceTesting/jazzer#fuzzed-data-provider).

Fuzzing for entire web apps with no manual setup is a feature of Code Intelligence's commercial fuzzing platform [CI Fuzz](https://www.code-intelligence.com).

**Reply by hector-rojas-sci** [reply]
Thank you so much, I have done my test using the first option.
